modules/opencensus/src/main/java/org/apache/ignite/spi/metric/opencensus/OpenCensusMetricExporterSpi.java

```diff
@@ -118,6 +120,26 @@
      */
     private Map<String, Measure> measures = new HashMap<>();
 
+    /** Cached histogram metrics intervals names. */
+    private Map<T2<String, long[]>, String[]> histogramNames = new HashMap<>();
+
+    /** Create histogram interval names. */
+    private static final Function<T2<String, long[]>, String[]> CREATE_INTERVAL_NAMES = k -> {
+        String histogramName = k.get1();
+        long[] bounds = k.get2();
+
+        String[] names = new String[bounds.length + 1];
+
+        for (int i = 0; i < bounds.length + 1; i++) {
```

Comment:
`i < bounds.length + 1` should be replaced with `i <= bounds.length`

Reply:
Done.